atsign-foundation/atsign.dev-3.0
# Move `reference/` above `contribute/`

**- What I did**
1. Moved `REFERENCE` above `CONTRIBUTE TO THIS SITE` using weights (see Fig A)
2. Extracted SDK services (see Fig B) (I personally think this is better and easier for the user to quickly find information on the packages. Also makes it clear that there are some core packages). (This is a rough draft for now, just considering general layout)

**- How to verify it**
Fig A - weights of each dir's `_index.md`
```
// BEFORE
start/ "Getting Started" - 99
tutorials/ "Samples and Tutorials" - 98
sdk/ "SDK Reference" - 97
atplatform/ - 96
contribute/ - 95
reference/ "Reference " - 94

//AFTER
start/ "Getting Started" - 99
tutorials/ "Samples and Tutorials" - 98
sdk/ "SDK Reference" - 97
atplatform/ - 96
reference/ "Reference " - 95
contribute/ - 94
```

Fig B - extracted code labs into singles
- also note that reference is above contribute
![image](https://user-images.githubusercontent.com/79019866/173961765-186a3ecb-426b-4bc5-b4e3-7df4a1e62002.png)
